app/controllers/admin/tools_controller.rb

```diff
@@ -22,6 +22,27 @@ def bust_cache
       redirect_to admin_tools_path
     end
 
+    def feed_playground
+      return if params[:config].blank?
+
+      begin
+        config_json = JSON.parse(params[:config])
+        config = Articles::Feeds::VariantAssembler
+          .build_with(catalog: Articles::Feeds.lever_catalog, config: config_json, variant: "test")
+        @user = User.find_by(username: params[:username]) || current_user
+        @feed = Articles::Feeds::VariantQuery.new(
+          config: config,
+          user: @user,
+          number_of_articles: params[:number_of_articles] || 25,
+          page: 1,
+          tag: nil,
+        )
+        @articles = @feed.more_comments_minimal_weight_randomized.to_a
```

Comment:
The maintenance of this interface `@feed.more_comments_minimal_weight_randomized.to_a` can be dropped, but that's sort of out of scope of this PR. There's definitely code quality improvements in the core feed work to be done, but not within scope.